### PR TITLE
meta: Use the DAML SDK Docker container alongside CircleCI's stock Python container

### PR DESCRIPTION
**Pull request by da-tanabe**
Use the DAML SDK Docker container alongside CircleCI's stock Python container.

Also some minor code fixes to remove deprecation warnings.

This fully addresses #75.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -56,20 +62,34 @@ jobs:
...
-            - python/dist
+          working_directory: python
+          
 
 workflows:
   version: 2
   build_and_test:
     jobs:
-      - build:
+      - build: {}
```
Comment (jordan-da) [inline .circleci/config.yml:93]: i actually think this can be a list of strings or dicts

File: _build/daml-install [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+export PATH=$PATH:~/.daml/bin
+
+daml_bin=$(which daml)
```
Comment (jordan-da) [inline _build/daml-install:5]: i don't see you using this?

File: python/Makefile [hunk 3]
```diff
@@ -77,7 +77,12 @@ unit-test:
 
 .PHONY: integration-test
 integration-test:
-	cd ../_fixtures/src/post-office && $(python) integration-test.py
+	cd ../_fixtures/src/post-office && \
+	if [ -z "$${DAZL_TEST_DAML_LEDGER_URL}" ]; then \
+		$(python) integration-test.py; \
+	else \
+		$(python) integration-test.py --url $${DAZL_TEST_DAML_LEDGER_URL}; \
+	fi
```
Comment (jordan-da) [inline python/Makefile:85]: avoid the inline if with 

```
$(python) integration-test.py $(if $(DAZL_TEST_DAML_LEDGER_URL),--url $(DAZL_TEST_DAML_LEDGER_URL))
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -1,11 +1,16 @@
 version: 2
 jobs:
   build:
-    working_directory: ~/dazl
     docker:
-      - image: rappdw/docker-java-python
+      - image: cimg/python:3.6.11
+
+      # Start an SDK image alongside the tests for purposes of running the Sandbox
+      - image: digitalasset/daml-sdk:1.3.0
```
Comment (jordan-da) [inline .circleci/config.yml:8]: do they have a `latest` tag? for this, it maybe better to use

File: .circleci/config.yml [hunk 5]
```diff
@@ -56,20 +62,34 @@ jobs:
...
+      - run:
+          name: Install poetry and jq
+          command: |
+            curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python && \
+            source $HOME/.poetry/env && \
+            pip install pipenv && \
+            poetry config virtualenvs.in-project true && \
+            curl -LO https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 && \
+            chmod 755 jq-linux64 && \
+            mv jq-linux64 /usr/bin/jq
```
Comment (jordan-da) [inline .circleci/config.yml:81]: could be cached

File: .circleci/config.yml (hunk 4 above)
Comment (da-tanabe) [reply .circleci/config.yml:8]: Good call—unfortunately no `latest` tag though.

File: python/Makefile (hunk 3 above)
Comment (da-tanabe) [reply python/Makefile]: Thanks for that! I knew there was a less stupid way to do this… :-)

File: _build/daml-install (hunk 2 above)
Comment (da-tanabe) [reply _build/daml-install]: Whoops—vestigial, and removed.

File: .circleci/config.yml (hunk 5 above)
Comment (da-tanabe) [reply .circleci/config.yml:81]: Indeed…maybe for a later date. I also want to better understand Blackduck's dependencies (and maybe this all goes into its own Docker image as discussed).

File: .circleci/config.yml (hunk 1 above)
Comment (da-tanabe) [reply .circleci/config.yml]: thx changed

**Pull request merged**